### Setup

In [7]:
import sys
sys.path.append('../')
del sys

%reload_ext autoreload
%autoreload 2
%load_ext tensorboard

from toolbox.paths import TENSORBOARD_LOGS_BASELINES_PATH, TENSORBOARD_LOGS_MODELS_PATH

root = "../"

The tensorboard extension is already loaded. To reload it, use:
  %reload_ext tensorboard


### Parameters

In [8]:
baselines = True
experiment_name = "context_free_0"

### Display the TensorBoard

In [9]:
tensorboard_logs_path = TENSORBOARD_LOGS_BASELINES_PATH if baselines \
    else TENSORBOARD_LOGS_MODELS_PATH

logdir = root + tensorboard_logs_path + experiment_name

In [10]:
%tensorboard --logdir $logdir

Reusing TensorBoard on port 6006 (pid 8944), started 20:40:54 ago. (Use '!kill 8944' to kill it.)

If an error occur with port 6006 (default one), in a shell, run:
    - lsof -i:6006
    (list the processes running on it with their PIDs)
    - kill -9 PID
    (to kill the corresponding process, for each process)